package.json

```diff
@@ -88,6 +88,7 @@
     "@babel/preset-typescript": "^7.12.7",
     "@babel/register": "^7.12.10",
     "@babel/runtime": "^7.12.5",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.4.3",
```

Comment:
hmm - this was last published 2 years ago an is 'experimental'... I guess if it works, it works, but it doesn't exactly inspire confidence.

Reply:
@dbkr new version is in progress, so don't worry about it. The old HMR was hell to setup in this project and didn't work at all.

Reply:
Oh, I see - they're working on 0.5. Plus I guess 0.4.3 has 2M+ downloads. I guess let's try it then.